src/main.ts

```diff
@@ -229,11 +229,13 @@ export class Purchases {
    * @param rcPackage - The package you want to purchase. Obtained from {@link Purchases.getOfferings}.
    * @param customerEmail - The email of the user. If null, RevenueCat will ask the customer for their email.
    * @param htmlTarget - The HTML element where the billing view should be added. If null, a new div will be created at the root of the page and appended to the body.
+   @param purchaseOptionId - The ID for a specific purchase option. Used to purchase trials and other offers. Obtained from {@link Purchases.getOfferings}.
    * @returns The customer info after the purchase is completed successfuly.
    * @throws {@link PurchasesError} if there is an error while performing the purchase. If the {@link PurchasesError.errorCode} is {@link ErrorCode.UserCancelledError}, the user cancelled the purchase.
    */
   public purchasePackage(
     rcPackage: Package,
+    purchaseOptionId: string, // WARNING: This is a breaking change
```

Comment:
Can I find a way to make this breaking change less impactful?

Or, if I have to make a breaking change, can I make it so that it reduces chances of a breaking change in the future? (Maybe using an object for the parameters with optional fields)